web/stations/stations-europe.js

```diff
@@ -1799,7 +1821,7 @@ var stations = {
         {
             name: "Radio Dimensione Suono (RDS)",
             logo: "stations/images-europe/italy/Radio Dimensione Suono (RDS).png",
-            url: "https://stream.rds.radio/rds/app128.stream/playlist.m3u8"
```

Comment:
Didn't notice it before the PR. Don't know what happened here but it seems like it still works(?)